docs/modules/operation/pages/telemetryd/protocols/jti.adoc

```diff
@@ -1,7 +1,7 @@
 
 = Junos Telemetry Interface
 
-The Junos Telemetry Interface (JTI) allows users to push operational statistics asynchronously to {page-component-title}.
+The Junos Telemetry Interface (JTI) lets users push operational statistics asynchronously to {page-component-title}.
```

Comment:
Standardizing how the other protocols use "you" instead of "users"
```suggestion
The Junos Telemetry Interface (JTI) lets you push operational statistics asynchronously to {page-component-title}.
```